src/tools/pontoon/teams_projects.md

```diff
@@ -2,69 +2,74 @@
 
 <!-- toc -->
 
-The [Teams page](https://pontoon.mozilla.org/teams/) lists all locales that are enabled in Pontoon. From here you can access a specific [Team page](#team-page), where you’ll get a list of all projects enabled for the requested locale. Selecting a project from the *Team page* will take you to what is called the [Localization page](#localization-page).
+The [Teams page](https://pontoon.mozilla.org/teams/) lists all locales that are enabled in Pontoon. From here specific [Team pages](#team-page) can be accessed, which contains a list of all projects enabled for the requested locale. Selecting a project from the *Team page* leads to what is called the [Localization page](#localization-page).
 
-From each locale’s *Team page* it’s also possible to [request a project](#requesting-a-project), that is asking to add that locale to a project already available in Pontoon, not asking to support localization for a brand new project.
+From each locale’s *Team page* it’s also possible to [Request a project](#requesting-a-project). Note that this is a request to add the locale to a project already available in Pontoon, not for requesting a brand new project.
```

Comment:
```suggestion
From each locale’s *Team page* it’s also possible to [Request a project](#requesting-a-project). Note that this is a request to add the locale to a project already available in Pontoon, it can’t be used for requesting a brand new project.
```

The alternative would be be "to add…, not to request".